pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[tool.ruff]
```

Comment:
All that for a single test file?

Sure, I guess...

Reply:
fwiw, I was thinking about putting the `tsconfig.json` from cockpit into some common location (`pkg/lib` probably) where we could use it from other projects... I wonder if we want to do similar for our ruff config.

Reply:
We already do this in podman, machines, with different selects so that will be some work.